libs/ui/src/radio_group/index.tsx

```diff
@@ -16,26 +16,44 @@ export interface RadioGroupProps<T extends RadioGroupOptionId> {
    * still allowing it to be assigned to the control for screen readers.
    */
   label: string;
+  /** @default 1 */
+  numColumns?: number;
   onChange: (newId: T) => void;
   options: ReadonlyArray<RadioGroupOption<T>>;
   selectedOptionId?: T;
 }
 
 const OuterContainer = styled.fieldset.attrs({ role: 'radiogroup' })`
-  display: inline-block;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
 `;
 
 const LabelContainer = styled.legend`
   display: block;
   margin-bottom: 0.5rem;
 `;
 
-const OptionsContainer = styled.span`
+interface OptionsContainerProps {
+  numColumns: number;
+}
+
+const OPTION_SPACING_REM = 0.5;
+
+const OptionsContainer = styled.span<OptionsContainerProps>`
+  align-items: stretch;
   border-radius: 0.25rem;
   display: flex;
-  flex-direction: column;
-  gap: 1rem;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: ${OPTION_SPACING_REM}rem;
+  height: 100%;
   margin-bottom: 0.35rem;
+
+  & > * {
+    flex-grow: 1;
+    width: calc(100% / ${(p) => p.numColumns} - ${OPTION_SPACING_REM}rem);
```

Comment:
Doesn't particularly matter for a two column case, but should this `OPTION_SPACING_REM` be multiplied by `(numColumns - 1)`?

Reply:
Thanks for catching that! Coming back with fresh eyes, I'm realising I should just switch over to a grid instead and avoid having to do these calculations manually